server/model to be implemented/fishSupplies.js

```diff
@@ -1,4 +1,81 @@
-// Fish_Supplies.js
-
 // These never change? So no need for a template?
-// - Type of Thing (food, toy, eggs, health booster, etc?)
\ No newline at end of file
+// - Type of Thing (food, toy, eggs, health booster, etc?)
+
+
+module.exports = function(sequilize, DataTypes) {
```

Comment:
I see what you're going for here - there's a better way to execute this though.  :)  

Let's talk - we'll need to build a SQL populate script (it's common to have a script that populates your development database so you have some test data).